integration/test/clusterstate/main_test.go

```diff
@@ -104,13 +106,24 @@ func init() {
 // TestMain allows us to have common setup and teardown steps that are run
 // once for all the tests https://golang.org/pkg/testing/#hdr-Main.
 func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	var err error
+	var v int
+
 	{
-		c := setup.Config{
+		c := e2esetup.Config{
 			AWSClient: c,
 			Guest:     g,
 			Host:      h,
 		}
 
-		setup.Setup(m, c)
+		err = e2esetup.Setup(ctx, m, c)
+		if err != nil {
+			l.Log(fmt.Sprintf("%#v\n", err))
```

Comment:
Uses the logger we already created.

Reply:
I think it would be good to use LogCtx even though we don't pass any loggin info atm

Reply:
Also I'd prefer to keep the structure, i.e. fields like level, message and stack.

Reply:
I didn't want to change too much at once. But this makes total sense and we can do it now. I'll update.